src/scripts/app.ts

```diff
@@ -1303,8 +1303,16 @@ export class ComfyApp {
     const executionStore = useExecutionStore()
     executionStore.lastNodeErrors = null
 
-    let comfyOrgAuthToken =
-      (await useFirebaseAuthStore().getIdToken()) ?? undefined
+    let comfyOrgAuthToken = await useFirebaseAuthStore()
+      .getIdToken()
+      .catch((error: unknown) => {
+        useDialogService().showErrorDialog(error, {
+          title: t('errorDialog.defaultTitle'),
+          reportType: 'authenticationError'
+        })
+        console.error(error)
+      })
```

Comment:
Could this happen inside the `getIdToken` action?

Reply:
You can also return `undefined` from the catch if you want to eliminate the nullish coalescing on 1332 (though I think it'd be undefined by default, right?)